utilities/testing/holohub.ctest

```diff
@@ -0,0 +1,77 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
It's not immediately obvious how the `.ctest` files should be used, could you please add a brief description either as an inline header or in a `utilities/testing/README.md` file?

Based on first read I think these are:
- `holohub.container.ctest`: generic, can use with any individual application, intended to run in app container
- `holohub.ctest`: specific to our test machine configuration, tests multiple apps known to have matching environment requirements, targeting bare metal (or default HoloHub container?). Recommend that we keep this internal

Reply:
Following offline discussion I think we are aligned on removing this file as it's not currently exercised/tested, let me know if you disagree